notifier.rst

```diff
@@ -63,6 +63,7 @@ OvhCloud    ``symfony/ovhcloud-notifier``      ``ovhcloud://APPLICATION_KEY:APPL
 Sinch       ``symfony/sinch-notifier``         ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
 FreeMobile  ``symfony/free-mobile-notifier``   ``freemobile://LOGIN:PASSWORD@default?phone=PHONE``
 Smsapi      ``symfony/smsapi-notifier``        ``smsapi://TOKEN@default?from=FROM``
+Infobip     ``symfony/infobip-notifier``     ``infobip://TOKEN@default?from=FROM``
```

Comment:
We should sort them alphabetically in a separate PR.

Same for the mailer transports.

Are you open for a PR @jeremyFreeAgent 

And thanks for your contribution 🥳

Reply:
Done in #14211